src/Components/Organisms/FOPaperRow/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useRef, useState } from "react";
+import Box from "@mui/material/Box";
+import Paper from "@mui/material/Paper";
+
+import { FATypography } from "@Atoms";
+
+import { IFOPaperRowProps } from "./FOPaperRow.interface";
+
+export const FOPaperRow = (props: IFOPaperRowProps) => {
+  const { items, actions } = props;
+  const actionRef = useRef<HTMLDivElement>(null);
+
+  const [actionWidth, setActionWidth] = useState("");
+
+  useEffect(() => {
+    if (actionRef.current?.offsetWidth) {
+      setActionWidth(`${actionRef.current.offsetWidth}px`);
+    }
+  }, [actionRef.current?.offsetWidth]);
+
+  return (
+    <Paper className="flex-paper-2" sx={{ position: "relative" }}>
+      <Box
+        display="flex"
+        gap={3}
+        sx={{ width: `calc(100% - ${actionWidth})` }}
+        overflow="auto"
+      >
+        {items.map((item, idx) => {
+          return (
+            <Box key={idx}>
+              <Box mb={1}>
+                <FATypography
+                  testID="typography-paper-row-item-title"
+                  variant="overline"
+                  color="text.secondary"
+                >
+                  {item.title}
+                </FATypography>
+              </Box>
+              {typeof item.children === "string" ? (
+                <FATypography
+                  testID="typography-paper-row-item-children"
+                  variant="body1"
+                >
+                  {item.children}
+                </FATypography>
+              ) : (
+                <Box>{item.children}</Box>
+              )}
+            </Box>
+          );
+        })}
+      </Box>
+      {actions && (
+        <Box
```

Comment:
better use grid system. [here's](https://mui.com/components/grid/#variable-width-content) the reference